CHANGELOG.md

```diff
@@ -3,10 +3,24 @@
 
 ## (Unreleased) hipBLAS 0.54.0
 ### Added
+- added option to opt-in to use __half type for fp16 for c++ users who define HIPBLAS_USE_HIP_HALF
```

Comment:
Maybe make more explicit, this is "to use __half for hiblasHalf in the API" or some wording to indicate this is API change.

Reply:
Changed this, let me know if you have any other concerns.